src/openforms/forms/api/serializers/logic/action_serializers.py

```diff
@@ -72,6 +73,17 @@ class LogicFetchActionSerializer(serializers.Serializer):
     )
 
 
+class VariableMappingSerializer(serializers.Serializer):
+    form_variable = serializers.CharField(
+        required=True,
+        validators=[variable_key_validator],
+        help_text=_("Key of the form variable."),
+    )
+    dmn_variable = serializers.CharField(
+        required=True, help_text=_("DMN input expression.")
```

Comment:
I think "expression" is confusing here, since inside the DMN tables you can use expressions too.

Maybe:
```suggestion
        required=True, label=_("DMN input parameter name.")
```